docs/manual/scala/guide/cluster/PersistentEntityCassandra.md

```diff
@@ -68,8 +68,6 @@ lagom.persistence.read-side.cassandra {
 
 With these properties set to `false`, if the keyspaces or tables are missing at startup your service will log an error and fail to start.
 
-It is not possible to disable automatic creation of the offset store table at this time.
-
```

Comment:
Ah, good to know. Not sure why I haven't use "global find" to check this. I'll modify it now, thanks!